boot-qemu.py

```diff
@@ -286,6 +328,13 @@ def parse_arguments():
     if args.append:
         runner.cmdline += args.append
 
+    if args.efi:
+        runner.efi = runner.supports_efi()
+        if not runner.efi:
+            utils.yellow(
+                f"EFI boot requested on unsupported architecture ('{args.architecture}'), ignoring...",
```

Comment:
trailing comma :(

Reply:
Whoops :/ sorry about that:

cd70025276cf65bee01166d332c276909c279924